utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -206,3 +206,13 @@ annotation class EnableSpotlightUi
 
 /** Default value for the feature flag corresponding to [EnableSpotlightUi]. */
 const val ENABLE_SPOTLIGHT_UI_DEFAULT_VALUE = true
+
+/**
+ * Qualifier for the platform parameter that controls whether configuration change for Input
+ * Interactions.
```

Comment:
This could be clearer--it should describe what the feature actually is, e.g.: "...controls whether input interaction state is correctly retained across configuration changes."

Reply:
Updated Kdoc, thanks for you suggestion